orchestrator/gbt/src/client/cosmos_to_eth.rs

```diff
@@ -68,7 +68,8 @@ pub async fn cosmos_to_eth(
 
     match balance {
         Some(balance) => {
-            if balance.amount < amount.amount.clone() + bridge_fee.amount.clone() {
+            // TODO fix this `unwrap` along with the `exit`
```

Comment:
Failures from the checked add operation need to be handled gracefully in conjunction with the `exit` when we fix issue #37 .

Reply:
I've fixed the #37 here the PR. https://github.com/onomyprotocol/cosmos-gravity-bridge/pull/59
So it would be better if you could fix this `unwrap` here in that PR.